packages/@aws-cdk/aws-dynamodb-global/lib/lambda-global-dynamodb.ts

```diff
@@ -0,0 +1,59 @@
+import cfn = require("@aws-cdk/aws-cloudformation");
+import iam = require("@aws-cdk/aws-iam");
+import lambda = require("@aws-cdk/aws-lambda");
+import cdk = require("@aws-cdk/cdk");
+import fs = require("fs");
+import path = require("path");
+import { DynamoDBGlobalStackProps } from "./aws-dynamodb-global";
+
+/**
+ * A stack that will make a Lambda that will launch a lambda to glue
+ * together all the DynamoDB tables into a global table
+ */
+export class LambdaGlobalDynamoDBMaker extends cdk.Stack {
+    /**
+     * The singleton Lambda function that will connect all the DynamoDB tables together into a global table
+     */
+    public lambdaFunction: lambda.SingletonFunction;
+
+    /**
+     * The content of the lambdaFunction (python3.7 using boto3)
+     */
+    public lambdaFunctionContent: string;
+
+    /**
+     * The CloudFormation CustomResource that will manage the lambda
+     */
+    public customResource: cfn.CustomResource;
+
+    constructor(scope: cdk.Construct, id: string, props: DynamoDBGlobalStackProps) {
+        super(scope, id, props);
+        const codeLocation = path.resolve(__dirname, "lambda", "handler.js.lambda");
+        this.lambdaFunctionContent = fs.readFileSync(codeLocation, "utf8");
+        this.lambdaFunction = new lambda.SingletonFunction(this, id + "-SingletonLambda", {
+            code: new lambda.InlineCode(this.lambdaFunctionContent),
+            description: "Lambda to make DynamoDB a global table",
+            handler: "index.handler",
+            runtime: lambda.Runtime.NodeJS810,
+            timeout: 300,
+            uuid: "D38B65A6-6B54-4FB6-9BAD-9CD40A6DAC12",
+        });
+        this.lambdaFunction.addToRolePolicy(new iam.PolicyStatement()
+            .allow()
+            .addAllResources()
+            .addAction("iam:CreateServiceLinkedRole")
+            .addAction("application-autoscaling:DeleteScalingPolicy")
+            .addAction("application-autoscaling:DeregisterScalableTarget")
+            .addAction("dynamodb:CreateGlobalTable")
+            .addAction("dynamodb:DescribeLimits")
+            .addAction("dynamodb:UpdateGlobalTable"));
```

Comment:
The `dynamodb:*` permissions should be offered as static methods, either in this package or `@aws-cdk/aws-dynamodb`.